<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Krudio</class>
 <widget class="QMainWindow" name="Krudio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>328</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>328</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>328</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Krudio</string>
  </property>
  <property name="windowIcon">
   <iconset theme="radio">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>182</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::ExtendedSelection</enum>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>179</y>
      <width>250</width>
      <height>150</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>false</bool>
    </property>
    <property name="documentMode">
     <bool>false</bool>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset theme="arrow-down">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QPushButton" name="prevPlay">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>80</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-skip-backward">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pausePlay">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-playback-start">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPlay">
      <property name="geometry">
       <rect>
        <x>115</x>
        <y>80</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-skip-forward">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
     <widget class="QPushButton" name="pausePause">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="media-playback-pause">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="waitMinute">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>123</y>
        <width>211</width>
        <height>16</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Подождите минутку...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QSlider" name="volumeChange">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>211</width>
        <height>21</height>
       </rect>
      </property>
      <property name="singleStep">
       <number>1</number>
      </property>
      <property name="pageStep">
       <number>10</number>
      </property>
      <property name="value">
       <number>50</number>
      </property>
      <property name="tracking">
       <bool>true</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>5</x>
        <y>3</y>
        <width>201</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string>Название трека</string>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="wordWrap">
       <bool>true</bool>
      </property>
      <property name="openExternalLinks">
       <bool>true</bool>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::NoTextInteraction</set>
      </property>
     </widget>
     <widget class="QPushButton" name="nextPlay_2">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>80</y>
        <width>40</width>
        <height>30</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="search">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
     <zorder>pausePause</zorder>
     <zorder>prevPlay</zorder>
     <zorder>pausePlay</zorder>
     <zorder>nextPlay</zorder>
     <zorder>label_5</zorder>
     <zorder>volumeChange</zorder>
     <zorder>nextPlay_2</zorder>
     <zorder>waitMinute</zorder>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset theme="document-edit">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string notr="true"/>
     </attribute>
     <widget class="QLineEdit" name="nameStations">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string comment="Name">Название</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="urlStations">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>191</width>
        <height>29</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>61</y>
        <width>55</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>URL</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>115</y>
        <width>85</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="dialog-ok-apply">
        <normaloff>.</normaloff>.</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>117</x>
        <y>115</y>
        <width>85</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset theme="edit-delete">
        <normaloff>.</normaloff>.</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="icon">
      <iconset theme="configure">
       <normaloff>.</normaloff>.</iconset>
     </attribute>
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Тема иконок</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>191</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Размер иконок</string>
      </property>
     </widget>
     <widget class="QSlider" name="horizontalSlider">
      <property name="geometry">
       <rect>
        <x>13</x>
        <y>110</y>
        <width>188</width>
        <height>21</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="invertedAppearance">
       <bool>false</bool>
      </property>
      <property name="invertedControls">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Светлая</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radioButton_2">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>40</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <property name="text">
       <string>Темная</string>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
      <property name="autoExclusive">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>label_3</zorder>
     <zorder>label_4</zorder>
     <zorder>horizontalSlider</zorder>
     <zorder>radioButton_2</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton</zorder>
     <zorder>radioButton_2</zorder>
    </widget>
   </widget>
  </widget>
  <action name="action">
   <property name="text">
    <string>О программе</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
